src/UIState/SeeVersion.cpp

```diff
@@ -0,0 +1,14 @@
+/**
+ * SeeVersion.cpp
+ */
+
+#include "SeeVersion.h"
+
+#include "../TankControllerLib.h"
```

Comment:
Does it work without the "../" prefix?

Reply:
Yup, it does work!

Reply:
At least locally

Reply:
I miss read that, my bad. I'll check that really quick.

Reply:
It works